modules/programs/borgmatic.nix

```
@@ -56,10 +76,20 @@ let
         };
 
         repositories = mkOption {
-          type = types.listOf types.str;
-          description = "Paths to repositories.";
-          example =
-            literalExpression ''["ssh://myuser@myrepo.myserver.com/./repo"]'';
+          type = types.listOf (types.either types.str repositoryOption);
+          description = "Paths to key/value pairs of repositories.";
```

"Paths to key/value pairs" doesn't mean anything to me. Maybe something like:

```suggestion
          description = "List of local or remote repositories with paths and optional labels.";
```